Test/FMI3/fmu_dummy/fmu3_model.c

```diff
@@ -110,35 +110,44 @@ fmi3Status fmi_set_debug_logging(fmi3Instance instance, fmi3Boolean loggingOn)
     } else {
         inst->loggingOn = loggingOn;
         return fmi3OK;
-    }
-}
+    }}
 
-fmi3Status fmi_get_float64(fmi3Instance instance, const fmi3ValueReference vr[], size_t nvr, fmi3Float64 value[], size_t nValues)
+fmi3Status fmi_get_float64(fmi3Instance instance, const fmi3ValueReference valueReferences[],
+    size_t nValueReferences, fmi3Float64 values[], size_t nValues)
```

Comment:
```suggestion
        size_t nValueReferences, fmi3Float64 values[], size_t nValues)
```

double indent on line continuation